livesync_citadel/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+graft indico_livesync_citadel/migrations
+graft indico_livesync_citadel/models
```

Comment:
since the plugin has some translatable strings, we also need

```
indico_livesync_citadel/translations
```

Reply:
`models`, on the other hand, is a python package and thus does not need to be listed there ;)

Reply:
true, but this is some leftovers from penelope's plugin. I didn't carefully take a look at it.